nodejs-ipaddr.js/nodejs-ipaddr.js.spec

```diff
@@ -0,0 +1,60 @@
+%global npm_name ipaddr.js
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 1.2.0
+Release: 1%{?dist}
+Summary: A library for manipulating IPv4 and IPv6 addresses in JavaScript
+License: MIT
+URL: https://github.com/whitequark/ipaddr.js 
+Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
+ExclusiveArch: %{nodejs_arches} noarch
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
+%package doc
+Summary: Documentation for nodejs-%{npm_name}
+Group: Documentation
+Requires: nodejs-%{npm_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+This package contains documentation for nodejs-%{npm_name}
+
+%prep
+%setup -q -n package
+
+%build
+%nodejs_symlink_deps --build
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr .npmignore .travis.yml Cakefile LICENSE README.md bower.json ipaddr.min.js lib package.json src test %{buildroot}%{nodejs_sitelib}/%{npm_name}
```

Comment:
`src` and `test` can probably be left out, one's the CoffeeScript source and tests are unlikely to be used here.